packages/grafana-data/src/utils/logs.test.ts

```diff
@@ -1,4 +1,5 @@
-import { LogLevel } from '../types/logs';
```

Comment:
I have moved `logs sorting functions` and associated tests from grafana `core` to `grafana/data` so we can use it in `grafana/ui` components such as `LogsRow`.